tests/integration/base.py

```diff
@@ -165,13 +165,12 @@ def packages_config(self):
     @property
     def selectors_config(self):
         return None
-
-    def unique_schema(self):
-        schema = self.schema
-
-        to_return = "{}_{}".format(self.prefix, schema)
-
-        return to_return.lower()
```

Comment:
I removed lower convert to test schema upper case

Reply:
I created the PR about this separately

https://github.com/databricks/dbt-databricks/pull/541